### PR TITLE
#2113 Hovered structure gets into the saved molecule

### LINKED ISSUE #2113: Hovered structure gets into the saved molecule
**Steps to Reproduce**
1. Click the tool _Single Bond_
2. Click on the canvas to draw a structure
3. Click the tool _Selection_, and select the structure drawn on step 2
4. Press Ctrl+x to cut it
5. Press Ctrl+v, but do not click on the canvas (means you haven't yet decided to paste it)
6. Press Ctrl+s

**Actual behavior**
You can see the data of the structure(which you haven't pasted on the canvas) shows on the preview text

**Expected behavior**
The data of the structure(which you haven't pasted on the canvas) doesn't show on the preview text

**Screenshots**
ketcher-paste-bug-10

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version 109.0.5414.120 (Official Build) (64-bit)

**Ketcher version** [e.g. v2.4.2]. 
v2.7.0 rc.10

**Additional context**
_Originally posted by @KonstantinEpam23 in https://github.com/epam/ketcher/issues/2079#issuecomment-1378765574_

### PR DESCRIPTION
**Pull request by yuleicul**
Resolves #2113

### CODE REVIEW

File: packages/ketcher-react/src/script/ui/state/hotkeys.ts [hunk 1]
```diff
@@ -42,9 +42,9 @@ export function initKeydownListener(element) {
   }
 }
 
-function removeNotRenderedStruct(actionTool, event, dispatch) {
-  const { code, metaKey } = event
-  if (actionTool.tool === 'paste' && code === 'KeyS' && metaKey) {
+function removeNotRenderedStruct(actionTool, group, dispatch) {
+  const effectedTools = ['paste', 'template']
+  if (effectedTools.includes(actionTool.tool) && group?.includes('save')) {
     dispatch({
```
Comment (yuleicul) [inline packages/ketcher-react/src/script/ui/state/hotkeys.ts:48]: If I understand correctly, `code === 'KeyS' && metaKey` is supposed to mean *the currently pressed keys are Ctrl+s*, right? I didn't know what `metaKey` is, and here is what I learned from [MDN](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey):
> On Macintosh keyboards, this is the ⌘ Command key. At least as of Firefox 48, the ⊞ Windows key is no longer considered the "Meta" key. KeyboardEvent.metaKey is false when the ⊞ Windows key is pressed.

So it seems that `metaKey` makes no sense, right? If my understanding is wrong, please let me know, thank you!

Comment (ansivgit) [reply packages/ketcher-react/src/script/ui/state/hotkeys.ts:48]: I used external Win KB for Mac and key ⊞ works for me as original key ⌘ Command on Mac KB. May be it helps

**Comment by KonstantinEpam23** [comment]
@St-Permiakov @yuleicul we might need to get back to this after we introduce a common tool for paste and templates, I will leave it in open state for now, thx!

**Review by St-Permiakov — APPROVED** [review]

File: packages/ketcher-react/src/script/ui/state/hotkeys.ts [hunk 2]
```diff
@@ -42,9 +42,9 @@ export function initKeydownListener(element) {
   }
 }
 
-function removeNotRenderedStruct(actionTool, event, dispatch) {
-  const { code, metaKey } = event
-  if (actionTool.tool === 'paste' && code === 'KeyS' && metaKey) {
+function removeNotRenderedStruct(actionTool, group, dispatch) {
+  const effectedTools = ['paste', 'template']
```
Comment (Nitvex) [inline packages/ketcher-react/src/script/ui/state/hotkeys.ts:46]: Guess, you've meant that
```suggestion
  const affectedTools = ['paste', 'template']
```

Comment (yuleicul) [reply packages/ketcher-react/src/script/ui/state/hotkeys.ts]: Thank you. Updated.

**Review by Konstantin1996 — APPROVED** [review]

**Review by Konstantin1996 — APPROVED** [review]

**Pull request merged**